samples/create_deployment.js

```diff
@@ -0,0 +1,59 @@
+// Copyright 2020, Google LLC.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+/**
+ * Create a Game Servers Deployment.
+ * @param {string} projectId string project identifier
+ * @param {string} deploymentId unique identifier for the new Game Server Deployment
+ */
+function main(
+  projectId = 'YOUR_PROJECT_ID',
+  deploymentId = 'DEPLOYMENT_ID'
+) {
+  // [START cloud_game_servers_create_deployment]
+  /**
+   * TODO(developer): Uncomment these variables before running the sample.
+   */
+  // const projectId = 'Your Google Cloud Project ID';
+  // const deploymentId = 'A unique ID for the Game Server Deployment';
+  const {
+    GameServerDeploymentsServiceClient,
+  } = require('@google-cloud/game-servers');
+
+  const client = new GameServerDeploymentsServiceClient();
+
+  async function createGameServerDeployment() {
+    const request = {
+        parent: `projects/${projectId}/locations/global`,
+        deploymentId: deploymentId,
+        gameServerDeployment: {
+            description: "nodejs test deployment",
+        },
+    };
+
+    const [operation] = await client.createGameServerDeployment(request);
+    const [result] = await operation.promise();
+
+    console.log('Game Server Deployment created:');
+    console.log(`\t Deployment name: ${result.name}`);
+    console.log(`\t Deployment description: ${result.description}`);
+    
+    // [END cloud_game_servers_create_deployment]
```

Comment:
This end tag would have to go down under `createGameServerDeployment();`, otherwise the copied code from cloudsite won't run

Reply:
(here and below)

Reply:
Done!

Reply:
Can you please explain the process? How are the tags used?

Reply:
Sure!  So after this lands on master, you could decide to use these samples on cloud.google.com.  You could go to any page in cloudsite in piper, and use a special tag to include code from github.  See this page as an example:
https://cloud.google.com/translate/docs/advanced/translating-text-v3#translating_text

The code you put between those region tags directly correlate to what gets embedded in cloudsite.  They also get tracked at http://devrel/snippets which has all kinds of neat usage data.

Reply:
Thanks. This is useful. I'll fix it for other samples as well in a separate PR.